dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.webBundleSecurity.jakarta-1.0.feature

```diff
@@ -0,0 +1,16 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = io.openliberty.webBundleSecurity.jakarta-1.0
```

Comment:
This name looks odd to me because there is no such feature as `jakarta-1.0`. I suggest calling this `io.openliberty.webBundleSecurity1.0-servlet5.0`.

Reply:
If we need to rename I suggest `io.openliberty.restHandler.jakarta` as before or perhaps `io.openliberty.restHandler1.0.jakarta`

Reply:
Either of those are fine

Reply:
Changed filename to `io.openliberty.webBundleSecurity1.0.jakarta`. Thanks!